### PR TITLE
remove x86 builds from CI

### PR DESCRIPTION
**Pull request by nilsnolde**
remove x86 builds from CI (32 bit CI build via #2791 in future)

What issue is this PR targeting? If there is no issue that addresses the problem, please open a corresponding issue and link it here.

## Tasklist

 - [ ] Add tests
 - [ ] Add #fixes with the issue number that this PR addresses
 - [ ] Update the docs with any new request parameters or changes to behavior described
 - [ ] Update the [changelog](CHANGELOG.md)

## Requirements / Relations

 Link any requirements here. Other pull requests this PR is based on?

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 * **Removed**
    * REMOVED: Remove Node bindings. [#2502](https://github.com/valhalla/valhalla/pull/2502)
    * REMOVED: appveyor builds. [#2544](https://github.com/valhalla/valhalla/issues/2544)
+   * REMOVED: Removed x86 CI builds. [#2792](https://github.com/valhalla/valhalla/issues/2792)
```
Comment (nilsnolde) [inline CHANGELOG.md:5]: ugh.. not `issues`, `pulls`.. makes me want to do this: https://github.com/valhalla/valhalla/pull/2763#issuecomment-756804568.. all these wasted CI hours for dumb commits like this

**Comment by kevinkreiser** [comment]
@nilsnolde what about the cigwin build?

**Comment by nilsnolde** [comment]
yeah, true.. should I just include that in this PR? (changing to PR)?

**Comment by kevinkreiser** [comment]
@nilsnolde yeah sure please include it!

File: CHANGELOG.md (hunk 1 above)
Comment (kevinkreiser) [reply CHANGELOG.md:5]: looks like the one above is the same too, should be: https://github.com/valhalla/valhalla/pull/2550

**Review by kevinkreiser — APPROVED** [review]

**Pull request merged**